modules/deployment/src/main/java/org/eclipse/xpanse/modules/deployment/deployers/terraform/TerraformBootDeployment.java

```diff
@@ -112,9 +119,22 @@ public DeployValidationResult validate(Ocl ocl) {
                 terraformApi.validateWithScripts(getDeployWithScriptsRequest(ocl));
         DeployValidationResult result = new DeployValidationResult();
         BeanUtils.copyProperties(validate, result);
+        try {
```

Comment:
in the line 121, we are already filling the values in `result`. Do we need still need this?

Reply:
The type of the filed `diagnostics` in these two class are different. The method `BeanUtils.copyProperties(validate, result)` could not copy this filed. I removed the ` copyProperties`  method.